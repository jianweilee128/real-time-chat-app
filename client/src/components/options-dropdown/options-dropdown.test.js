import React from "react";
import { shallow } from "enzyme";

import OptionsDropdown from "./options-dropdown.component";

describe("OptionsDropdown component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<OptionsDropdown.WrappedComponent />);
  });

  it("should render OptionsDropdown component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
