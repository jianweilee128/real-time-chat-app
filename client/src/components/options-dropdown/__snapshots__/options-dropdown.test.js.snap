// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptionsDropdown component should render OptionsDropdown component 1`] = `ShallowWrapper {}`;
